### DISCUSSION TITLE
Bestway powered dreamstream

### DISCUSSION THREAD

**Discussion by bartbakels** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#287] [General]
Does anybody have success connecting the dreamstream spas? I know the use the same pumps as bestway.

this is the type P06461

![5F1133E6-42B9-4B9B-826D-29D550160EE7](https://user-images.githubusercontent.com/6642211/170883057-d2aa6b63-2f28-407b-99a0-1f7d9e679ab9.jpeg)
![EB1B0C29-4E64-4E64-A8C5-BB42D767568A](https://user-images.githubusercontent.com/6642211/170883067-6570de4f-3f60-4a85-a88b-a4f8c52e675a.jpeg)

**Comment by bartbakels** [comment]
Seems to be 6 wires from the panel.

im gonna try the miami config when i have all components ;)

now with picture , however display pcb says 2019

![BECF04DA-A6E3-47F6-A1D1-A1F5D95D5218](https://user-images.githubusercontent.com/6642211/170889848-82bdee84-de6a-4832-9425-615c42e7a3bc.jpeg)

**Reply by visualapproach** [reply]
Sounds reasonable. Good luck

**Reply by bartbakels** [reply]
Yeah but pcb says 2019…

**Reply by visualapproach** [reply]
Just try the pre2021 model if not working.

**Reply by bartbakels** [reply]
I can confirm this model ( bought at action NL) is also working with 6 wires miami 2021 and the 3 resistors

bart

**Reply by visualapproach** [reply]
Super! Thank you for sharing and have a nice day
